.github/workflows/tox.yml

```diff
@@ -66,6 +66,8 @@ jobs:
         exclude:
           - python: "3.10"
             django: "main"
+          - python: "3.11"
+            django: "main"
```

Comment:
Can we update the tox.ini as well so that local tests are kept in sync with the CI.